python/cudf/cudf/tests/test_orc.py

```diff
@@ -795,3 +795,22 @@ def test_orc_string_stream_offset_issue():
     df.to_orc(buffer)
 
     assert_eq(df, cudf.read_orc(buffer))
+
+
+def test_orc_reader_decimal(datadir):
+    path = datadir / "TestOrcFile.decimal.orc"
```

Comment:
Side note - we don't need to use the repo file now that decimal support is done in ORC, but I think it's fine since there's another test that uses this file.